### PR TITLE
Update documentation for Harman Kardon component

### PR DESCRIPTION
**Pull request by Devqon**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23569

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/harman_kardon_avr.markdown [hunk 1]
```diff
@@ -28,7 +28,16 @@ To add a Harman Kardon Network Receiver to your installation, add the following
 # Example configuration.yaml entry
 media_player:
```
Comment (klaasnicolaas) [inline source/_components/harman_kardon_avr.markdown]: Only required variables in the first example

File: source/_components/harman_kardon_avr.markdown [hunk 2]
```diff
@@ -10,8 +10,8 @@ footer: true
 logo: harman_kardon.png
 ha_category: Media Player
 ha_iot_class: Local Polling
-ha_release: 0.85
-redirect_from:
+ha_release: 0.85.0
```
Comment (klaasnicolaas) [inline source/_components/harman_kardon_avr.markdown]: Please undo this

File: source/_components/harman_kardon_avr.markdown [hunk 3]
```diff
@@ -10,8 +10,8 @@ footer: true
...
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/harman_kardon_avr.markdown]: Please undo this

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**